lib/ecto/association.ex

```diff
@@ -35,7 +35,7 @@ defmodule Ecto.Association do
                required(:cardinality) => :one | :many,
                required(:relationship) => :parent | :child,
                required(:owner) => atom,
-               required(:owner_key) => atom,
+               required(:owner_key) => atom | list(atom),
```

Comment:
We should probably normalize it to always be a list? 🤔

Reply:
This will make sure we consider owner_key everywhere possible. Possibly the same change needs to be done for related_key!

Reply:
Worth noting that this changing to a list caused a regression in absinthe’s dataloader in my project. I’ve [set up an initial PR](https://github.com/absinthe-graphql/dataloader/pull/138) there to address it for when this feature lands

Reply:
__probably__ should be kept as `atom() | list(atom)`, then? To avoid breaking compatibility?